### DISCUSSION TITLE
Postmortem boards

### DISCUSSION THREAD

**Discussion by TareqK** [Leantime/leantime#1358] [Ideas]
Currently, we only support team retrospective boards - which are excellent for reviews. However I feel like having incident/post mortem boards could be valuable, as it allows direct followup on live incidents and shorter cycles to review them.

**Comment by marcelfolaron** [comment]
Hm, in my experience postmortems tend to be rather long. Maybe a template for Docs would be better?

**Reply by TareqK** [reply]
My line of thought is a column per postmortem question based on the Google SRE framework, and to have them centralized.

Maybe even tied back to milestones or OKRs at some point
